InvenTree/templates/InvenTree/settings/user.html

```diff
@@ -248,7 +248,11 @@ <h4>{% trans "Active Sessions" %}</h4>
         {% for object in session_list %}
             <tr {% if object.session_key == session_key %}class="active"{% endif %}>
             <td>{{ object.ip }}</td>
+            {% if object.user_agent or object.device %}
```

Comment:
Question: Did that raise an error?

Reply:
See my comment here : https://github.com/jazzband/django-user-sessions/pull/87

Reply:
Intresting, that is a new one.